tests/framework/CodeInterfaceTests/test_Dymola_code_interface_timedep.xml

```diff
@@ -119,20 +121,28 @@
        <Input>G_a, G_b, G_c</Input>
        <Output>OutputPlaceHolder</Output>
     </PointSet>
-    <HistorySet name="BBHistorySet_out">
-       <Input>hstart, G_vect</Input>
-       <Output>v,flying,impact</Output>
-       <options>
-           <pivotParameter>Time</pivotParameter>
-       </options>
-     </HistorySet>
     <HistorySet name="CreateHIST_out">
       <Input>G_a, G_b, G_c</Input>
       <Output>G_vect</Output>
        <options>
-           <pivotParameter>Time</pivotParameter>
+           <pivotParameter>TimeFirstModel</pivotParameter>
        </options>
     </HistorySet>
+    <!--
```

Comment:
Do we want to leave this old history set stuff as reference?  Just confirming.

Reply:
yes I would like to keep this for reference